doc/source/examples/analysis/alignment_and_rms/pairwise_rmsd.ipynb

```diff
@@ -185,6 +196,93 @@
     "plt.colorbar(label='RMSD (Angstrom)')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pairwise RMSD between two trajectories"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Calculating the 2D RMSD between two trajectories is a bit more finicky; `DistanceMatrix` can only calculate the RMSD of a trajectory to itself. Instead, we do it the long way by simply calculating the RMSD of each frame in the second trajectory, to each frame in the first trajectory.\n",
+    "\n",
+    "First we set up a numpy array to store our results. To start off, it's all zeros."
```

Comment:
"it's all zeros" -> "we populate it with zero values" might be more easy to understand?

Reply:
Might be worth being a bit more descriptive here, 2D with lengths equal to each frame of the trajectory.